src/java/voldemort/server/protocol/vold/ClientRequestHandler.java

```diff
@@ -0,0 +1,138 @@
+package voldemort.server.protocol.vold;
```

Comment:
This will cause lots of change and lots of merge conflict, I have too much code in motion. So would avoid this one. The protocol is still named as Voldemort-native in most of the code, so vold seems better to me than the native.